### PR TITLE
CICD

### PR DESCRIPTION
**Pull request by EgorKrasno**
- Created a CICD pipeline for AWS
- Swapped to yaml instead of properties files
- Created yaml (properties) for tests

### CODE REVIEW

File: elasticbeanstalk/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    image: "egorkra/nineline:latest"
+    ports:
+      - "80:8080"
+    restart: "always"
+    environment:
+#      PASSWORD: ${ADMIN}
+#      STRIPE: ${STRIPE}
+#      JWT: ${JWT_SECRET}
+#      AWSKEY: ${AWS_KEY}
+#      AWSSECRET: ${AWS_SECRET}
```
Comment (jfarrin73) [inline elasticbeanstalk/docker-compose.yaml:13]: delete these

File: src/test/java/com/teamb/nineline/controller/RequestControllerTest.java [hunk 2]
```diff
@@ -188,7 +188,7 @@ public void updateRequestStatusById() throws Exception {
     @Rollback
     public void catchRequestExistsException() throws Exception{
 
-        MockHttpServletRequestBuilder request = get("/api/request/" + 1);
+        MockHttpServletRequestBuilder request = get("/api/request/" + 10);
```
Comment (jfarrin73) [inline src/test/java/com/teamb/nineline/controller/RequestControllerTest.java:191]: I don't think we want this change pushed, right?

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
+/*
```
Comment (jfarrin73) [inline .mvn/wrapper/MavenWrapperDownloader.java:1]: Should this file be in gitignore?

File: elasticbeanstalk/docker-compose.yaml (hunk 1 above)
Comment (EgorKrasno) [reply elasticbeanstalk/docker-compose.yaml:13]: but what if we want to integrate stripe later?

Comment (jfarrin73) [reply elasticbeanstalk/docker-compose.yaml:13]: 💸💸💸

**Pull request merged**